modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2389,4 +2421,10 @@ private Collection<UUID> collectTableIndexes(UUID tableId) {
 
         return indexIds;
     }
+
+    private static void closeTracker(@Nullable PendingComparableValuesTracker<?> tracker) {
```

Comment:
We already have this method `IgniteUtils#closeAllManually`, but it doesn't fit because it (`closeAllManually`) throws an exception that `PendingComparableValuesTracker#close` doesn't have.